lib/modules/datasource/index.spec.ts

```diff
@@ -212,6 +230,12 @@ describe('modules/datasource/index', () => {
       expect(res).toMatchObject({ releases: [{ version: '0.0.1' }] });
     });
 
+    it('defaultRegistryUrls function works', async () => {
+      datasources.set(datasource, new DummyDatasource2());
+      const res = await getPkgReleases({ datasource, depName });
+      expect(res).toMatchObject({ releases: [{ version: '1.2.3' }] });
```

Comment:
```suggestion
      expect(res).toMatchObject({ releases: [{ version: '1.2.3' }], registryUrl: 'https://reg1.com' });
```

Shouldn't we also validate `registryUrl` ?

Reply:
What does validate mean here, I am not sure?